sql/aiven_extras--1.0.0--1.0.1.sql

```diff
@@ -1 +0,0 @@
-DROP FUNCTION IF EXISTS pg_list_all_subscriptions;
```

Comment:
Meh, this doesn't work, we still need this!

Reply:
Also: the tag for this is 1.0, not 1.0.0. No clue if this makes a difference, but I think it will (pg gets the version from aiven-extras.control were it probably was '1.0.0'.